libs/fluid-elements/tab-group/src/lib/tab/tab.ts

```diff
@@ -0,0 +1,163 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  LitElement,
+  CSSResult,
+  property,
+  TemplateResult,
+  html,
+} from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map';
+import styles from './tab.scss';
+
+let _unique = 0;
+
+/** Custom event implementation that fires when a tab is clicked providing the active tab id  */
+export class FluidTabActivatedEvent extends CustomEvent<any> {
+  constructor(public activeTab: string) {
+    super('tabActivated', { bubbles: true, composed: true });
+  }
+}
+
+export class FluidTabDisabledEvent extends CustomEvent<any> {
+  constructor(public disableTab: string) {
+    super('disabled', { bubbles: true, composed: true });
+  }
+}
+
+/**
+ * This is a experimental version of the tab component
+ * It registers itself as `fluid-tab` custom element.
```

Comment:
```suggestion
 * It registers itself as `fluid-tab` custom element.
 * @slot - Default slot to provide a label for the tab.
```

Reply:
Added that too. What's with github. I guess because that specific line didn't change